### PR TITLE
7.0.0 release

### PR DESCRIPTION
**Pull request by wyardley**
This release includes some breaking(ish) changes, so we're going to do a major bump.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,57 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not impact the functionality of the module.
 
-## [v6.0.0](https://github.com/voxpupuli/puppet-rabbitmq/tree/v6.0.0) (2017-09-06)
+## [v6.0.1-rc0](https://github.com/voxpupuli/puppet-rabbitmq/tree/v6.0.1-rc0) (2017-09-13)
```
Comment (alexjfisher) [inline CHANGELOG.md]: Run changelog task *after* updating the version. :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,57 @@ All notable changes to this project will be documented in this file.
...
+- Move examples and params to puppet strings style docs [\#562](https://github.com/voxpupuli/puppet-rabbitmq/pull/562) ([wyardley](https://github.com/wyardley))
+
+**Fixed bugs:**
+
+- Idempotency problems with rabbitmq 3.6.5 and puppet 4.8.2 [\#618](https://github.com/voxpupuli/puppet-rabbitmq/issues/618)
+- Enabling new plugin fails [\#566](https://github.com/voxpupuli/puppet-rabbitmq/issues/566)
+- Rabbitmq\_user\_permissions parser fails for empty string [\#172](https://github.com/voxpupuli/puppet-rabbitmq/issues/172)
+- Error on using puppet resource rabbitmq\_user [\#147](https://github.com/voxpupuli/puppet-rabbitmq/issues/147)
+- Resolve issue with "puppet resource rabbitmq\_user" failing \(\#147\) [\#629](https://github.com/voxpupuli/puppet-rabbitmq/pull/629) ([wyardley](https://github.com/wyardley))
+- BREAKING: Ensure python package \(adds manage\_python option\) [\#623](https://github.com/voxpupuli/puppet-rabbitmq/pull/623) ([wyardley](https://github.com/wyardley))
```
Comment (alexjfisher) [inline CHANGELOG.md]: Maybe tempted to label the BREAKING changes 'enhancement's so that they feature prominently in the changelog?

Comment (wyardley) [reply CHANGELOG.md]: I think we should keep them as bug only; the ones that are not already tagged as enhancements are not really enhancements, so I don't think we should tweak it just for the higher placement in the changelog (though I like the idea of the breaking changes section that @bastelfreak was working on).

Comment (wyardley) [reply CHANGELOG.md]: Also, I don't think the "breaking" ones for this release are super breaking, most of them should not affect users of recent RabbitMQ versions IMHO.

Comment (wyardley) [reply CHANGELOG.md]: Added enhancement to the most important one.

**Comment by wyardley** [comment]
@alexjfisher version was fixed yesterday, and I cleaned up the changelog. Just re-ran to catch #634 as a bug in the Changelog. Can you re-review?

**Pull request merged**